test/integration/analytics_lakehouse/analytics_lakehouse_test.go

```diff
@@ -35,13 +34,13 @@ func TestAnalyticsLakehouse(t *testing.T) {
 	dwh.DefineVerify(func(assert *assert.Assertions) {
 		dwh.DefaultVerify(assert)
 
-		projectID := dwh.GetTFSetupStringOutput("project_id")
-		bucket := dwh.GetStringOutput("raw_bucket")
+		// projectID := dwh.GetTFSetupStringOutput("project_id")
+		// bucket := dwh.GetStringOutput("raw_bucket")
 
-		bucketOP := gcloud.Runf(t, "storage buckets describe gs://%s --project %s", bucket, projectID)
-		assert.Equal("US-CENTRAL1", bucketOP.Get("location").String(), "should be in us-central1")
-		assert.Equal("STANDARD", bucketOP.Get("storageClass").String(), "should have standard storageClass")
-		//TODO: Add additional asserts for other resources
+		// bucketOP := gcloud.Runf(t, "storage buckets describe gs://%s --project %s", bucket, projectID)
+		// assert.Equal("US-CENTRAL1", bucketOP.Get("location").String(), "should be in us-central1")
+		// assert.Equal("STANDARD", bucketOP.Get("storageClass").String(), "should have standard storageClass")
+		// TODO: Add additional asserts for other resources
```

Comment:
nit: let's remove these and open an tracking issue to add more assertions in the test?

Reply:
done